packages/poaps/README.md

```diff
@@ -2,49 +2,83 @@
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
 
-`@poap-xyz/poaps` is a package to fetch information about POAPs tokens, their on-chain data, their related drop and metadata.
+`@poap-xyz/poaps` is a JavaScript library providing an interface for interacting with POAPs (Proof of Attendance Protocol tokens), allowing you to fetch information about POAP tokens, their on-chain data, their related drop and metadata, all in a simplified manner.
```

Comment:
Remove the `Proof of Attendance Protocol` (in all the READMEs) as we are trying to avoid it